docs/admin/access.rst

```diff
@@ -168,15 +168,15 @@ to a containing project as well.
 
 .. note::
 
-   Only first rule which is set is being used. So if you set all
+   Only the first rule will be used. So if you set all of
    :guilabel:`Component list`, :guilabel:`Components` and :guilabel:`Project`,
-   only :guilabel:`Component list` is being applied.
+   only :guilabel:`Component list` is the only one applied.
 
-Additional step is performed if checking permission for the translation:
+An additional step is performed if checking permission for the translation:
 
 
-4. :guilabel:`Languages` are matches against scope of translations, if not set, this matches no
```

Comment:
is matched against?
Can "if not set" be avoided?

Reply:
Group languages are matched against the translation languages. If no group languages are set, it matches none languages (this can be still useful to grant access to a project or component, but not to actual translations.